zone/zonedb.cpp

```diff
@@ -687,7 +687,8 @@ bool ZoneDatabase::LoadCharacterData(uint32 character_id, PlayerProfile_Struct*
 		pp->ldon_points_ruj = Strings::ToInt(row[r]); r++;								 // "ldon_points_ruj,           "
 		pp->ldon_points_tak = Strings::ToInt(row[r]); r++;								 // "ldon_points_tak,           "
 		pp->ldon_points_available = Strings::ToInt(row[r]); r++;							 // "ldon_points_available,     "
-		pp->tribute_time_remaining = Strings::ToInt(row[r]); r++;							 // "tribute_time_remaining,    "
+		//pp->tribute_time_remaining = Strings::ToInt(row[r]); r++;							 // "tribute_time_remaining,    "
+		pp->tribute_time_remaining = atoi(row[r]); r++;							 // "tribute_time_remaining,    "
```

Comment:
Why is this changed?

Reply:
the Strings:ToInt was causing an exception error when the value was 4294967295, which is in my character_data table.  It was not causing a crash, though I noticed it when I was debugging some of my code.  I tested outside of eqemu, and also found it gave the exception.  That said, I have returned it to the previous value.

Reply:
Maybe `Strings::ToUnsignedInt` if `tribute_time_remaining` is `uint32`?

Reply:
I changed to Unsigned and all seems to be well in my tests.  I will update if you folks are ok with that.

Reply:
Sounds good.

Reply:
Sounds good.  Pushed an update for this.